testing/test_error_diffs.py

```diff
@@ -0,0 +1,255 @@
+"""
+Tests and examples for correct "+/-" usage in error diffs.
+
+See https://github.com/pytest-dev/pytest/issues/3333 for details.
+
+"""
+import pytest
+
+
+TESTCASES = [
```

Comment:
it might be helpful, to switch from comments to pytest.param(..., id="cmp_foo_one_item_diff") or have a mapping of them in the declaration simply to align test case ids with test case meaning

Reply:
Uh, i haven’t used that before and [these docs](https://docs.pytest.org/en/latest/parametrize.html) don’t show anything about `parma(id=...)`. Could you please give me a short example?